### PR TITLE
Made it more sensible, man.

### PR DESCRIPTION
**Pull request by varshiths**

### CODE REVIEW

**Comment by varshiths** [comment]
Comment here.
Discussion of the pull request.

File: yo [hunk 1]
```diff
@@ -2,9 +2,14 @@ Moshi Moshi..!!
...
+
 
 Yoroshiku onegaishimasu. 
 
+^ Orgasm, not onegaishimasu. Haha.
+
 
 Adding rubbish... xD
 
+^ Publishing rubbish.
```
Comment (harshithxD) [inline yo]: -_-

**Comment by varshiths** [comment]
What was 'close' and 'reopen'?

**Comment by harshithxD** [comment]
You can close a pull request if you don't want to merge it. :-1:

**Comment by varshiths** [comment]
And you can reopen it

**Comment by varshiths** [comment]
Don't reply. Die, bitch.

**Pull request merged**